### ISSUE TITLE
I change network to 352*352,the train result is very chaotic

### ISSUE THREAD

**Issue by matakk** [AlexeyAB/darknet#198]
Hi ，I use yolov2 to detect object .I change to 352 so speed up the detect on every  frame.
my config is heare:
thanks!

[net]
batch=64
subdivisions=8
height=352
width=352
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.0001
max_batches = 70000
policy=steps
steps=100,30000,40000
scales=10,.1,.1

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

#######

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[route]
layers=-9

[reorg]
stride=2

[route]
layers=-1,-3

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=45
activation=linear

[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=4
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=1

**Comment by AlexeyAB** [comment]
> the train result is very chaotic

What does it mean?

CFG-file looks correct.

**Comment by matakk** [comment]
![default](https://user-images.githubusercontent.com/6842094/30551168-34f043cc-9ccc-11e7-9417-301dbcacfa80.png)
This is my result . I don't known why show like this.
Thanks!!!

**Comment by AlexeyAB** [comment]
1. Do you use my fork or original fork?
2. Do you train and detect using Yolo on Linux or Windows?
3. Did you train using the same cfg-file as for detection?
4. What command line did you use to train and detect?
5. What can you see, if you just change width=416 height=416 in cfg-file and try to detect without re-training?

**Comment by matakk** [comment]
Hi:
1:use your fork ,lastest 
2:on windows
3:yes ,same cfg ,just change the batch param.
4:train commandline: 
darknet.exe detector train data/veh3p_352.data data/veh3p_352.cfg darknet19_448.conv.23
detection is this fork ,I have warrper this in our lib.And I test 416 network is work.
5:I test 416 on this train weights,still show like this result.

Thanks!

**Comment by matakk** [comment]
my cfg is different  with train detect ,The filter is not same (3 or 4 )
Sorry weast your time!
Thansk again!!
